tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapEntryListenerTest.java

```diff
@@ -59,176 +73,200 @@
-                            eventCount.localEvictCount.getAndIncrement();
+                            eventCount.evictCount.getAndIncrement();
                         }
                         map.evict(key);
                     } finally {
                         map.unlock(key);
                     }
-                } else if (chance < removeProb + evictProb + writeProb) {
-                    Object o = map.remove(key);
-                    if (o != null) {
-                        eventCount.localRemoveCount.getAndIncrement();
+                    break;
+                case REMOVE:
+                    String oldValue = map.remove(key);
+                    if (oldValue != null) {
+                        eventCount.removeCount.getAndIncrement();
                     }
-                } else if (chance < deleteProb + removeProb + evictProb + writeProb) {
+                    break;
+                case DELETE:
                     map.lock(key);
                     try {
                         if (map.containsKey(key)) {
-                            eventCount.localRemoveCount.getAndIncrement();
+                            eventCount.removeCount.getAndIncrement();
                         }
                         map.delete(key);
                     } finally {
                         map.unlock(key);
                     }
-                }
+                    break;
+                default:
+                    throw new UnsupportedOperationException();
             }
-            IList results = targetInstance.getList(basename + "eventCount");
-            results.add(eventCount);
         }
-    }
 
-    @Verify(global = true)
-    public void globalVerify() throws Exception {
-        IList<EntryListenerImpl> listeners = targetInstance.getList(basename + "listeners");
-        for (int i = 0; i < listeners.size() - 1; i++) {
-            EntryListenerImpl a = listeners.get(i);
-            EntryListenerImpl b = listeners.get(i + 1);
-            assertEquals(basename + ": not same amount of event in all listeners", a, b);
+        @Override
+        protected void afterRun() {
+            eventCounts.add(eventCount);
         }
-    }
 
-    @Verify(global = false)
-    public void verify() throws Exception {
-        IList<EventCount> eventCounts = targetInstance.getList(basename + "eventCount");
-        EventCount total = new EventCount();
-        for (EventCount c : eventCounts) {
-            total.add(c);
-        }
-        total.waiteWhileListenerEventsIncrease(listener, 10);
-
-        log.info(basename + ": add = " + total.localAddCount.get() + " " + listener.addCount.get());
-        log.info(basename + ": update = " + total.localUpdateCount.get() + " " + listener.updateCount.get());
-        log.info(basename + ": remove = " + total.localRemoveCount.get() + " " + listener.removeCount.get());
-        log.info(basename + ": evict = " + total.localEvictCount.get() + " " + listener.evictCount.get());
```

Comment:
this was useful info to see. when the test fails,  do we still print the individual counts some how ?